modules/tensor_mechanics/doc/content/source/materials/lagrangian/ComputeStVenantKirchhoffStress.md

```diff
@@ -0,0 +1,46 @@
+# ComputeStVenantKirchhoffStress
```

Comment:
Years ago I attempted to implement such a model, leading to `ADComputeGreenLagrangeStrain` and `modules/tensor_mechanics/examples/hyper_elastic_test.i`. I should compare that to the new stuff.

Reply:
What confuses me about what you did there is that while a St. Venant Kirchhoff material is "standard linear elasticity using the Green-Lagrange strain" it produces the 2nd Pioloa-Kirchhoff stress, not the Cauchy stress which is what I think you're doing in this example.  Though maybe I'm missing something.

But that's the basic idea -- produce some stress measure directly from a large-deformation strain measure.